Conceptpower+Spring/src/main/java/edu/asu/conceptpower/root/DatabaseClient.java

```diff
@@ -296,6 +302,84 @@ public boolean match(ConceptList dict) {
      * lang.String)
      */
     @Override
+    public List<ConceptEntry> getAllEntriesFromList(String listname, int page, int pageSize, final String sortBy,
+            final int sortDirection) {
+        ConceptEntry entry = new ConceptEntry();
+        entry.setConceptList(listname);
+
+        Query wordNetQuery = wordnetCacheClient.query();
```

Comment:
why do you query the wordnet cache if you get all entries in a list here? wordnet concepts are by default in the wordnet list and this list is not accessible through the concept list webpage.